modules/tide_content_collection/src/SearchApiIndexHelper.php

```diff
@@ -296,6 +296,24 @@ public function buildEntityReferenceFieldFilter(IndexInterface $index, string $f
     return NULL;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getEntityReferenceFieldInfo(IndexInterface $index, string $field_id) : ?array {
+    $reference_fields = $this->extractIndexEntityReferenceFields($index);
+    if (!isset($reference_fields[$field_id])) {
+      return NULL;
+    }
+
+    try {
+      return $reference_fields[$field_id];
```

Comment:
@joshua-salsadigital what could possibly cause an exception here?

Reply:
Corrected the logic,